build_image.sh

```diff
@@ -21,7 +24,18 @@ make clean
 make
 
 BUILD_DATE=$(date +'%Y%m%d%H%M%S')
-REPO_NAME="openebs/cstor-base"
+if [ "${ARCH}" = "x86_64" ]; then
+	REPO_NAME="openebs/cstor-base"
+	DOCKERFILE_BASE="Dockerfile.base"
+	DOCKERFILE="Dockerfile"
+elif [ "${ARCH}" = "aarch64" ]; then
+	REPO_NAME="openebs/cstor-base-arm64"
+	DOCKERFILE_BASE="Dockerfile.arm64.base"
+	DOCKERFILE="Dockerfile.arm64"
+else
+	echo "Unusable architecture: ${ARCH}"
```

Comment:
Probably say -- "${ARCH} is not supported"

Reply:
The message I followed this https://github.com/openebs/maya/blob/5ed267c117b7a5e293ec9d822db154782a9fdbc6/buildscripts/build.sh#L52 ^^.